src/rosdiscover/cli.py

```diff
@@ -48,11 +50,37 @@ def launch(args) -> None:
 
 
 def generate_acme(args):
+    with open(args.il.name, 'r') as f: 
+        data = yaml.load(f, Loader=yaml.FullLoader) #generates dictionary from YAML
     """Generates an Acme description for a given roslaunch command."""
-    interpreter = _launch_config(args)
-    # nodes = [n.to_dict for n in interpreter.nodes]
 
-    acme_gen = AcmeGenerator(interpreter.nodes, args.acme, args.jar)
+    nodes = []
+    if args.il is not None:
```

Comment:
If `args.il` is `None`, the code should have thrown an exception when we try to `open` the file above. Maybe we want to add a check above?